doc-content/enterprise-only/project-data/config-project-settings-proc.adoc

```diff
@@ -0,0 +1,69 @@
+[id='config-project-settings']
+
+= Configuring project settings
+
+Starting in {PRODUCT} {PRODUCT_VERSION}, {CENTRAL} contains additional project setting categories in the new process designer.
+
+.Prerequisites
+* You have created a {CENTRAL} project.
+
+.Procedure
+. To access the project *Settings* tab, in {CENTRAL}, go to *Menu* -> *Design* -> *Projects*.
+. Click on the project name.
```

Comment:
It's never click on, woman! :-)

Reply:
STOP YELLING AT ME! I'm a woman under pressure. CLICK ON THAT!